test/parallel/test-dns.js

```diff
@@ -145,3 +145,19 @@ assert.doesNotThrow(function() {
     hints: dns.ADDRCONFIG | dns.V4MAPPED
   }, noop);
 });
+
+assert.throws(function() {
+  dns.lookupService('0.0.0.0');
+}, /Invalid arguments/);
+
+assert.throws(function() {
+  dns.lookupService('fasdfdsaf', 0, function() {});
```

Comment:
There is a `noop` function defined in this file that you can use instead of `function() {}`. Same comment for the other two uses.